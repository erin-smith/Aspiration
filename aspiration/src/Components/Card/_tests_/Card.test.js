import * as React from "react";
import Card from "../index";
import renderer from "react-test-renderer";

describe("Card", () => {
  test("Should match the snapshot", () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
