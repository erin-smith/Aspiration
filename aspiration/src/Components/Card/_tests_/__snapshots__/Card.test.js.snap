// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should match the snapshot 1`] = `
<div
  className="card mt-4"
>
  <div
    className="card-header"
  >
    <h3>
      <strong />
    </h3>
  </div>
  <div
    className="card-body"
  />
</div>
`;
